types/node/assert.d.ts

```diff
@@ -77,7 +77,28 @@ declare module "assert" {
         function match(value: string, regExp: RegExp, message?: string | Error): void;
         function doesNotMatch(value: string, regExp: RegExp, message?: string | Error): void;
 
-        const strict: typeof assert;
+        const strict: Omit<
+            typeof assert,
+            | 'strict'
+            | 'deepEqual'
+            | 'notDeepEqual'
+            | 'equal'
+            | 'notEqual'
+            | 'ok'
+            | 'strictEqual'
+            | 'deepStrictEqual'
+            | 'ifError'
```

Comment:
Only `deepEqual`, `notDeepEqual`, `equal` and `notEqual` are replaced: <https://github.com/nodejs/node/blob/e9639ee7e8705516e3b4b9ba7ee19d8e210afa9a/lib/assert.js#L934-L944>

```suggestion
            | 'deepEqual'
            | 'notDeepEqual'
            | 'equal'
            | 'notEqual'
```

Reply:
You have to do this to preserve the `asserts`, otherwise you'll get `Assertions require every name in the call target to be declared with an explicit type annotation`